### PR TITLE
CDAP-2224: measure type per each measure in cube query

### PR DESCRIPTION
**Pull request by abaranau**

### CODE REVIEW

**Comment by abaranau** [comment]
http://builds.cask.co/browse/CDAP-RBT234-1

File: cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricQueryParser.java [hunk 1]
```diff
@@ -154,7 +153,8 @@ static MetricDeleteQuery parseDelete(URI requestURI, String metricPrefix) throws
     builder.setMetricName(metricPrefix);
 
     MetricDataQuery query = builder.build();
-    return new MetricDeleteQuery(query.getStartTs(), query.getEndTs(), query.getMetricNames(), query.getSliceByTags());
+    return new MetricDeleteQuery(query.getStartTs(), query.getEndTs(),
+                                 query.getMetrics().keySet(), query.getSliceByTags());
```
Comment (gsps1) [inline cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricQueryParser.java]: this class should have been removed right ? as its for v2 API.

**Comment by gsps1** [comment]
LGTM :+1:

File: cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricQueryParser.java (hunk 1 above)
Comment (abaranau) [inline cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricQueryParser.java]: we do still deletion of metrics. not thru REST though

**Pull request merged**